scripts/high_level_motion_unit_test.py

```diff
@@ -0,0 +1,173 @@
+#! /usr/bin/env python
+import sys
+import rospy
+from iarc7_msgs.msg import OdometryArray
+import actionlib
+import tf2_ros
+from iarc7_motion.msg import QuadMoveGoal, QuadMoveAction
+from iarc7_safety.SafetyClient import SafetyClient
+
+def test():
+    safety_client = SafetyClient('high_level_motion_unit_test')
+    # Since this abstract is top level in the control chain there is no need to check
+    # for a safety state. We can also get away with not checking for a fatal state since
+    # all nodes below will shut down.
+    assert(safety_client.form_bond())
```

Comment:
No parentheses here, just `assert safety_client.form_bond()`, and same below.  I know there are parentheses after asserts all over this codebase, but there shouldn't be

Reply:
fixed